wandb/apis/public/api.py

```diff
@@ -29,9 +29,11 @@
 from wandb.apis.public.registries import Registries
 from wandb.apis.public.utils import (
     PathType,
+    check_server_feature_with_fallback,
     fetch_org_from_settings_or_entity,
     parse_org_from_registry_path,
 )
+from wandb.proto.v3.wandb_internal_pb2 import ServerFeature
```

Comment:
This should be agnostic to the proto version.

```suggestion
from wandb.proto.wandb_internal_pb2 import ServerFeature
```